src/daft-scheduler/src/scheduler.rs

```diff
@@ -259,6 +260,7 @@ fn physical_plan_to_partition_tasks(
     physical_plan: &PhysicalPlan,
     py: Python,
     psets: &HashMap<String, Vec<PyObject>>,
+    actor_pool_manager: &PyObject,
```

Comment:
Maybe a good idea to actually wrap this into a `ExecutionContext`? We could start using that to thread other things through as well (e.g. the execution config).

Not blocking for this PR, but could be great to formalize in the future.

Reply:
That might make sense. I'll try that

Reply:
After looking at it for a bit, I don't think it would be the best idea, since the execution config is actually put into the ray object store and sent to all of the tasks, whereas the actor pool manager does not need to leave the scheduler actor.